bundles/org.openhab.binding.sony/README-SimpleIp.md

```diff
@@ -0,0 +1,124 @@
+| inputsource       | R          | String       | The input source ("antenna"). See note 1 below                  |
+| input             | R          | String       | The input in the form of "xxxxyyyy" ("HDMI1"). See note 2 below |
+| picturemute       | R          | Switch       | Whether the picture is shown or not (muted)                     |
+| togglepicturemute | W          | Switch       | Toggles the picture mute                                        |
+| pip               | R          | Switch       | Enables or disabled picture-in-picture                          |
+| togglepip         | W          | Switch       | Toggles the picture-in-picture enabling                         |
+| togglepipposition | W          | Switch       | Toggles the picture-in-picture position                         |
+
+1.  The text of the input source is specific to the TV.  The documentation lists as valid dvbt, dvbc, dvbs, isdbt, isdbbs, isdbcs, antenna, cable, isdbgt.  However, "atsct" seems to be supported as well and others may be valid. 
+2.  The input can be either "TV" or "xxxxyyyy" where xxxx is the port name and yyyy is the port number.  Valid port names (case insensitive) are "hdmi", "scart", "composite", "component", "screen mirroring", and "pc rgb input".  The port number is dependent on how many ports the device supports.  Example: the X830 supports 4 hdmi ports - so "hdmi1", "hdmi2", "hdmi3" and "hdmi4" are all valid.
+
+## Full Example
+
+simpleip.Things:
+
+```
+sony:simpleip:home [ deviceAddress="192.168.1.72", commandsMapFile="braviaircodes.map", netInterface="eth0" ]
+```
+
+simpleip.items:
+
+```
+String Bravia_IR "IR [%s]" { channel="sony:simpleip:home:ir" }
+Switch Bravia_Power "Power [%s]" { channel="sony:simpleip:home:power" }
+Dimmer Bravia_Volume "Volume [%s]" { channel="sony:simpleip:home:volume" }
+Switch Bravia_AudioMute "Audio Mute [%s]" { channel="sony:simpleip:home:audiomute" }
+String Bravia_Channel "Channel [%s]" { channel="sony:simpleip:home:channel" }
+String Bravia_TripletChannel "Triplet Channel [%s]" { channel="sony:simpleip:home:tripletchannel" }
+String Bravia_InputSource "Input Source [%s]" { channel="sony:simpleip:home:inputsource" }
+String Bravia_Input "Input [%s]" { channel="sony:simpleip:home:input" }
+Switch Bravia_PictureMute "Picture Mute [%s]" { channel="sony:simpleip:home:picturemute" }
+Switch Bravia_TogglePictureMute "Toggle Picture Mute [%s]" { channel="sony:simpleip:home:togglepicturemute", autoupdate="false"  }
+Switch Bravia_Pip "PIP [%s]" { channel="sony:simpleip:home:pip" }
+Switch Bravia_TogglePip "Toggle PIP [%s]" { channel="sony:simpleip:home:togglepip", autoupdate="false" }
+Switch Bravia_TogglePipPosition "Toggle PIP Position [%s]" { channel="sony:simpleip:home:togglepipposition", autoupdate="false" }
+String Bravia_BroadcastAddress "Broadcast [%s]" { channel="sony:simpleip:home:broadcastaddress" }
+String Bravia_MacAddress "MAC [%s]" { channel="sony:simpleip:home:macaddress" }
```

Comment:
these two aren't mentioned in the channels table above

Reply:
Good catch - very, very old channels (that were removed - they are properties now). Removed